<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    font-family: century gothic;
    font-size: 18px;
}

QFrame {
    background: #333;
    border-radius: 15px;
}

QPushButton {
    background: #338508;
    color: white;
    border-radius: 15px;
}

QPushButton:hover {
    color: #333;
    background: #2e96dd;
    border-radius: 15px;
}

QToolButton {
    color: blue;
    background: #338508;
    border-radius: 60px;
}

/* Logo du haut */
QToolButton#toolButton {
    background: transparent;
    border: none;
}

QLabel {
    color: white;
}

QLineEdit {
    background: transparent;
    border: none;
    color: #717072;
    border-bottom: 1px solid #717072;
}

QRadioButton {
    color: white;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>551</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>340</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #2b2b2b; /* Couleur de fond */
    color: #ffffff;           /* Couleur du texte */

    padding: 5px;              /* Espacement interne */
    font-size: 16px;           /* Taille du texte */
}

QLineEdit:focus {
    border: 2px solid #00ff00; /* Couleur de bordure différente lorsque le champ est actif */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #2b2b2b; /* Couleur de fond */
    color: #ffffff;           /* Couleur du texte */

    padding: 5px;              /* Espacement interne */
    font-size: 16px;           /* Taille du texte */
}

QLineEdit:focus {
    border: 2px solid #00ff00; /* Couleur de bordure différente lorsque le champ est actif */
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-family: &quot;Verdana&quot;;
    font-size: 18px;
    font-weight: bold;
    color: #4CAF50; /* Texte vert */
    text-shadow: 2px 2px 5px #000000; /* Ombre noire */
}</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-family: &quot;Verdana&quot;;
    font-size: 18px;
    font-weight: bold;
    color: #4CAF50; /* Texte vert */
    text-shadow: 2px 2px 5px #000000; /* Ombre noire */
}</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
    font-family: &quot;Arial&quot;;
    font-size: 14px;
    color: #FFFFFF; /* Texte blanc */
}

QRadioButton::indicator {
    width: 15px;
    height: 15px;
    border: 1px solid #4CAF50; /* Bordure verte */
    background-color: #333333; /* Fond gris foncé */
    border-radius: 7px; /* Forme circulaire */
}

QRadioButton::indicator:checked {
    background-color: #4CAF50; /* Fond vert lorsqu'il est sélectionné */
    border: 1px solid #FFFFFF; /* Bordure blanche */
}
</string>
     </property>
     <property name="text">
      <string>show password</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/ecofit.png</normaloff>:/images/ecofit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
